_includes/card-purchase.md

```diff
@@ -296,6 +296,51 @@ Content-Type: application/json
 | └─➔&nbsp;`href`                   | `string`     | The target URI to perform the operation against.                                                                                                                                                                                                                                                                                                           |
 | └─➔&nbsp;`rel`                    | `string`     | The name of the relation the operation has to the current resource.                                                                                                                                                                                                                                                                                        |
 
+
+Customers should be able to set `paymentToken` to deleted if the consumer 
+deletes their agreements with their customer. 
```

Comment:
We need to explain how the operation that describes how to delete a token is found. What is its `rel` name? Please provide an example and update the `<OPERATION NAME>` in the description.

```suggestion
Payers should be able to delete payment tokens that are associated to
them. To delete a `paymentToken`, perform the `<OPERATION NAME>`
operation.

```